exporter/collector/integrationtest/testdata/fixtures/metrics/google_managed_prometheus_expect.json

```diff
@@ -641,6 +641,37 @@
               }
             }
           ]
+        },
+        {
+          "metric": {
+            "type": "prometheus.googleapis.com/target_info/gauge",
+            "labels": {
+              "instance": "10.92.5.2:15692",
```

Comment:
I don't think job/instance should be included in metric labels if they are already in resource labels.  IIRC, GMP will reject this point.

Reply:
From the spec (https://github.com/open-telemetry/opentelemetry-specification/blob/v1.16.0/specification/compatibility/prometheus_and_openmetrics.md#resource-attributes-1) I thought the point was that these would be included as metric labels?

Reply:
Prometheus has a "flat" label model.  There isn't such a thing as resource labels vs metric labels.  GMP hides the implementation from users by internally representing some labels (job, instance, cluster, namespace) as resource labels rather than metric labels, but they look the same to users at query time.

But in order for that to work, you can't have the same label key in both resource and metric labels in the metric sent to GCM.  Otherwise GMP doesn't know which value to use, so they just reject it.

Reply:
We need to add all _other_ resource labels (except those in the monitored resource) as metric labels on the metric (e.g. service.name -> service_name)

Reply:
done, also refactored how those labels are tracked a little too. maybe overthinking it